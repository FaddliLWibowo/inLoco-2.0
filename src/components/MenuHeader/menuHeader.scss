.menu-header {

    &--logo {
        height: 43px;
        margin: 20px 0 0 15px;
        width: 195px;
    }

    &--title {
        color: $sidebar-left-highlight-color;
        font: normal 16px sans-serif;
        margin: 20px 0 15px 15px;
        text-transform: uppercase;
    }

    &--caption {
        color: $menu-text-color;
        display: block;
        font-size: 80%;
        text-transform: initial;
    }
}
