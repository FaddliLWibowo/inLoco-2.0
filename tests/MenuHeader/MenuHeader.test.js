import React from 'react';
import MenuHeader from '../../src/components/MenuHeader/MenuHeader.js';

it('MenuHeader component renders correctly', () => {
    const menuHeader  = shallow(<MenuHeader/>);
    expect(menuHeader).toMatchSnapshot();
});
