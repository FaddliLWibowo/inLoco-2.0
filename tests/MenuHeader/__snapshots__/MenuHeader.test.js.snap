// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuHeader component renders correctly 1`] = `
<div
  className="menu-header"
>
  <img
    alt="InLoco"
    className="menu-header--logo"
    src={Object {}}
  />
  <a
    className="menu-header--button fa fa-times"
    role="button"
  />
  <h1
    className="menu-header--title"
  >
    Camadas de Exibição
    <small
      className="menu-header--caption"
    >
      Escolha uma categoria abaixo
    </small>
  </h1>
</div>
`;
